app/models/content.rb

```diff
@@ -18,7 +18,7 @@ def update_attributes_with_content(content)
       self.try("#{at}=", content[at])
     end
 
-    self.original_url = content['url']
+    self.href = content['url']
```

Comment:
When an audio file is changed (different original_url) via crier or cms announcement, the existing file is not updated, instead a new one is inserted, so it will continue to show the previously existing file for the same position until the new file has finished processing and deletes it. You can see this in the handlers and how they process changes to files.

So really, this should never result in an update.

That said, if logic elsewhere was changed, and a content was updated with a different url, then I do think this is the correct behavior. Without this, the original url would be changed, but because the file was already marked completed and a task to copy the (previous) original_url still exists, the new original_url audio won't actually get processed or updated at all - it will still act like it has processed b/c the prior url was processed and the task and status have not been reset.